cmd/mms/main.go

```diff
@@ -101,14 +107,14 @@ func main() {
 			{
 				Name:    "list-all",
 				Aliases: []string{"ls"},
-				Usage:   "List all the latest available events in the system",
+				Usage:   "List all the latest available events in the system.",
```

Comment:
Currently it's "all available", isn't it? Maybe we should allow to provide some "latest" specification also? Or for now drop it from the usage string?